packages/fxa-auth-server/scripts/delete-account.ts

```diff
@@ -118,6 +119,15 @@ DB.connect(config).then(async (db: any) => {
     Container.set(PayPalHelper, paypalHelper);
   }
 
+  const accountDeleteManager = new AccountDeleteManager({
+    fxaDb: db,
+    oauthDb: oauthDB,
+    push,
+    pushbox,
+    statsd,
+  } as any);
```

Comment:
Just curious, why is `as any` needed?

Reply:
To skip the type checking around the arguments in AccountDeleteManager, as it has type refinements on several of them that don't otherwise have real types (they come from loosely defined JS objects).